src/app/service/poap/SchedulePOAP.ts

```diff
@@ -51,14 +49,11 @@ const SchedulePOAP = async (ctx: CommandContext, guildMember: GuildMember, numbe
 		],
 	};
 	
-	if (isDmOn) {
-		await guildMember.send(msg1);
-		await ctx.send({ content: 'I just sent you a DM!', ephemeral: true });
-	} else if (ctx) {
+	if (ctx) {
```

Comment:
Do we need to keep this as an `if` statement, given that `ctx` is a required parameter for the whole function to run?

Reply:
I removed the if statement. I spoke to Brian about the chainID and duplicate values one-on-one. We agreed to keep it because I thought it was correct. Now, I think he was correct and I removed the Chain Id's. It will be more relevant when the WConnnect PR comes out though. It's ready as soon as this goes up!